pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1052,7 +1052,7 @@
         "4.0.0.0": "4.0.0",
         "4.1.0.0": "4.1.0",
         "4.1.1.0": "4.3.0",
-        "4.1.2.0": "4.4.0"
+        "4.2.0.0": "4.4.0"
```

Comment:
This file is maintained automatically by running msbuild /t:UpdatePackageIndex on the relevant package (and not running that when a change is needed results in build errors).  4.1.2 was never a stable/published version in the package, it was just a placeholder version number that was set after we shipped 4.1.1, so that we wouldn't inadvertently add to the same version.  But since new APIs are being added, which necessitates bumping the minor version, we changed the version number to 4.2.0 instead of to 4.1.2, and this reflects that.